docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -0,0 +1,494 @@
+
+#### Install EKS cluster
+
+Let's export environment variables that are needed for the EMR on EKS cluster setup. Please copy and paste commands into terminal for faster provisioning
+```
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+export EKS_CLUSTER_NAME="ack-emr-eks"
+export AWS_DEFAULT_REGION="us-west-2"
+export AWS_REGION="us-west-2"
+```
+We'll use eksctl to install EKS cluster.
+```
+eksctl create cluster -f - << EOF
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: ${EKS_CLUSTER_NAME}
+  region: ${AWS_DEFAULT_REGION}
+  version: "1.21"
+
+managedNodeGroups:
+  - instanceType: m5.xlarge
+    name: ${EKS_CLUSTER_NAME}-ng
+    desiredCapacity: 2
+
+iam:
+  withOIDC: true
+EOF
+```
+#### Create IAM Identity mapping
+We need to create emr-containers identity in EKS cluster so that EMR service has proper RBAC permissions needed to manage and run Spark jobs
+```
+export EMR_NAMESPACE=emr-ns
+echo "creating namespace for $SERVICE"
+kubectl create ns $EMR_NAMESPACE
+
+echo "creating iamidentitymapping using eksctl"
+eksctl create iamidentitymapping \
+   --cluster $EKS_CLUSTER_NAME \
+   --namespace $EMR_NAMESPACE \
+   --service-name "emr-containers"
+```
+**Expected outcome**
+```
+2022-08-26 09:07:40 [ℹ]  eksctl version 0.84.0
+2022-08-26 09:07:40 [ℹ]  using region us-west-2
+2022-08-26 09:07:42 [ℹ]  created "emr-ns:Role.rbac.authorization.k8s.io/emr-containers"
+2022-08-26 09:07:42 [ℹ]  created "emr-ns:RoleBinding.rbac.authorization.k8s.io/emr-containers"
+2022-08-26 09:07:42 [ℹ]  adding identity "arn:aws:iam::012345678910:role/AWSServiceRoleForAmazonEMRContainers" to auth ConfigMap
+```
+#### Install EMR on EKS controller
+Now we can go ahead and install EMR on EKS controller. First, let's export environment variables needed for setup
+```
+export SERVICE=emrcontainers
+export RELEASE_VERSION=v0.0.6
+export ACK_SYSTEM_NAMESPACE=ack-system
+```
+We cam use Helm for the installation
+```
+echo "installing ack-$SERVICE-controller"
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+helm install --create-namespace -n $ACK_SYSTEM_NAMESPACE ack-$SERVICE-controller \
+  oci://public.ecr.aws/aws-controllers-k8s/$SERVICE-chart --version=$RELEASE_VERSION --set=aws.region=$AWS_REGION
+```
+**Expected outcome**
+```
+NAME: ack-emrcontainers-controller
+LAST DEPLOYED: Fri Aug 26 09:05:08 2022
+NAMESPACE: ack-system
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+emrcontainers-chart has been installed.
+This chart deploys "public.ecr.aws/aws-controllers-k8s/emrcontainers-controller:v0.0.6".
+
+Check its status by running:
+  kubectl --namespace ack-system get pods -l "app.kubernetes.io/instance=ack-emrcontainers-controller"
+
+You are now able to create Amazon EMR on EKS (EMRContainers) resources!
+```
+#### Configure IRSA for emr on eks controller
```

Comment:
I think for this section, you can reference the OIDC configuration section of our docs - that's what [other tutorials](https://aws-controllers-k8s.github.io/community/docs/tutorials/memorydb-example/#configure-iam-permissions) have done. We would also typically do this before we install the controller, so that we can pass the role ARN as part of the Helm values file - that way, we don't have to modify the deployment manually.

Reply:
Gotcha. I'll do that as soon as this https://github.com/aws-controllers-k8s/community/issues/1468 is resolved